__tests__/app.test.js

```diff
@@ -46,3 +51,37 @@ describe("/api/topics", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id", () => {
+  test("GET 200: responds with correct article with all keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(1);
+        expect(typeof response.body.article.title).toBe("string");
+        expect(typeof response.body.article.topic).toBe("string");
+        expect(typeof response.body.article.author).toBe("string");
+        expect(typeof response.body.article.body).toBe("string");
+        expect(typeof response.body.article.created_at).toBe("string");
+        expect(typeof response.body.article.votes).toBe("number");
+        expect(typeof response.body.article.article_img_url).toBe("string");
```

Comment:
You can be more specific in this testing, and expect exact values from the test data rather than using `typeof`. This is because it's getting back a single object this time rather than an array, and we know what values are held in the article_id 1 object